src/pages/lessons/projects/5.mdx

```diff
@@ -23,274 +17,322 @@ import Layout from "../../../components/Layout";
+Expect somewhere from one to six hours to complete the lesson, depending on your
+prior experience and the need to grasp new concepts. To help you maintain a
+steady pace and encourage breaks for rejuvenation, you'll find our pomodoro
+timer ⌛ in the menu header. It'll remind you to "go touch grass" and appreciate
+nature's role in enhancing our well-being. 🌱 Relish your journey in development
+in a good state of mind and body.
 
 ## What are we building
 
-In the first in this track of tutorials, you created a basic smart contract using Remix. Then your shift to a dedicated developer's environment to create and deploy an NFT contract to an actual test network. In lesson 3 you added tiers to your NFTs. And you finally learnt to make your contracts airtight by building your own automated tests in the fourth in the series. Now we're going use all of this knowledge and connect it to a frontend for users to see and interact with.
-
-
-
-We'll create a basic **NFT minting application** that displays the three tiers of NFTs
-created in our smart contract and gives users a one-click option to mint them. Once minted, users will be redirected to a page that displays information of their mint. We'll need some extra magic to make that all happen. For handling wallet connections to our application, we'll be introducing you to *RainbowKit*. And using the provided *hooks* from *WAGMI*, we can check the connection to the wallet and call functions from our smart contract through the frontend. *RainbowKit* and *WAGMI* are just another couple of shining examples of open source public goods. We love open source at *Developer_DAO*. And public goods are gooood!
-
+In the first in this track of tutorials, you created a basic smart contract
+using Remix. Then your shift to a dedicated developer's environment to create
+and deploy an NFT contract to an actual test network. In lesson 3 you added
+tiers to your NFTs. And you finally learnt to make your contracts airtight by
+building your own automated tests in the fourth in the series. Now we're going
+use all of this knowledge and connect it to a frontend for users to see and
+interact with.
+
+We'll create a basic **NFT minting application** that displays the three tiers
+of NFTs created in our smart contract and gives users a one-click option to mint
+them. Once minted, users will be redirected to a page that displays information
+of their mint. We'll need some extra magic to make that all happen. For handling
+wallet connections to our application, we'll be introducing you to _RainbowKit_.
+And using the provided _hooks_ from _WAGMI_, we can check the connection to the
+wallet and call functions from our smart contract through the frontend.
+_RainbowKit_ and _WAGMI_ are just another couple of shining examples of open
+source public goods. We love open source at _Developer_DAO_. And public goods
+are gooood!
+
+- [RainbowKit](https://www.rainbowkit.com/docs/introduction) for out-of-the-box
```

Comment:
delete:
'[RainbowKit](https://www.rainbowkit.com/docs/introduction) for out-of-the-box wallet management (i.e. Handling wallet connections to our application).

[WAGMI](https://wagmi.sh/) for checking connection to a wallet and calling functions from our smart contract through the frontend using the provided hooks. */}'